man/systemd.exec.xml

```diff
@@ -1869,6 +1869,9 @@ SystemCallErrorNumber=EPERM</programlisting>
         variable definitions. The parser strips leading and trailing whitespace from the values of assignments, unless
         you use double quotes (").</para>
 
+        <para><ulink url="https://en.wikipedia.org/wiki/Escape_sequences_in_C#Table_of_escape_sequences">c-escapes</ulink>
+        are supported, so \t and \n can be used insert tabs and newlines within <varname>EnvironmentFile=</varname></para>
```

Comment:
Well, this mentions c escapes, but doesn't specify which ones can be used... As a reader of this, I could try \a or \b and be disappointed. I think it should explicitly say that control characters other than \n and \t are not supported.

Also please use `<literal>\n</literal>` and `<literal>\t</literal>`.